src/Propel/Generator/Behavior/NestedSet/NestedSetBehaviorObjectBuilderModifier.php

```diff
@@ -671,7 +678,7 @@ protected function addGetParent(string &$script): void
  * @param  ConnectionInterface \$con Connection to use.
```

Comment:
docblock should be also updated IMO

Reply:
True, but I think this should be a separate CS cleanup PR as outlined in beta ticket
Mixing all those into is out of scope.